src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/vs-2017.3.host.json

```diff
@@ -25,11 +25,19 @@
   ],
   "ports": [
     {
-      "name": "HttpPort",
+      "name": "kestrelHttpPort",
```

Comment:
The ports section from the host file isn't used by the VS 2022 any longer - this is when VS generated the ports - since the generator is built in you don't need to modify this.

In fact, I just noticed my PR for renaming the host files was not completed, you'll need to rebase when it completes.  I hadn't fully cleaned up the ports in that PR, since I wanted to be 100% sure it's not needed.  I'll likely send a PR next week to do more cleanup - we won't need the "supportedAuthentications" section in the host file any longer either due to recent dev17 changes.

Reply:
OK so that whole section will/can be removed from that file then?

Reply:
Yes